### ISSUE TITLE
508-defect-2: Conditional field labels must be descriptive

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#43249]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

Selecting "other" on the claimant information page causes a new field to appear dynamically asking the user to "please specify." However, it's not specific _what_ the user should be specifying. 
- For blind screen reader users, it may not be as clear that the other choice is directly linked with "please specify" as they may not be able to see the vertical blue line that connects the two fields together
- For Veterans with cognitive disabilities, a generic label with generic error messaging may also fail to communicate clearly what they should be doing even if the other field is nearby

I lack context to understand what "Claiming as a firm, corporation or state agency" alludes to and makes me wonder if that should be a more explicit option upfront.

## Acceptance Criteria

- [x] Additional fields are descriptively labeled e.g. `Describe the claimant's relationship to the deceased Veteran` 
- [x] Error messages are descriptive and follows VA error messaging guidance e.g. `Please describe a relationship`
- [x] Confirm and consider intent of "Claiming as a firm, corporation, or state agency"
- [ ] Copy has been reviewed with the content team @jsokohl 

## Steps to Recreate

1. [Go to the burials flow and click "start without signing in" ](https://staging.va.gov/burials-and-memorials/application/530/claimant-information)
2. Go to step 1 of 6: Claimant Information and select "other" for relationship to the deceased Veteran
3. Confirm label is "Please specify"
4. Go to step 2 of 6: Deceased Veteran Information on the date of death page and select "other" 
5. Confirm label is "Please specify"

## Proposed Solution (if known)

A more descriptive label will help screen reader users and users with cognitive disabilities understand what the additional field is for. I strongly recommend reaching out to the content team for more guidance, particularly on the "Claiming as a firm, corporation, or state agency" checkbox.

## Screenshots or Trace Logs

<img width="763" alt="Screen Shot 2022-06-23 at 10 32 51 AM" src="https://user-images.githubusercontent.com/14154792/175327906-76dc0f9d-4908-4428-af6e-02e15d44e824.png">
<img width="653" alt="Screen Shot 2022-06-23 at 10 32 41 AM" src="https://user-images.githubusercontent.com/14154792/175327984-19a26d08-ca67-4c42-a9f0-308be08aac38.png">
<img width="1063" alt="Screen Shot 2022-06-23 at 11 11 54 AM" src="https://user-images.githubusercontent.com/14154792/175333376-dc646614-b95d-4405-93f1-8e36318df82e.png">

**Comment by jsokohl** [comment]
Hey @joshkimux  and @skylerschain 
For the field label on step 1, you could change
`Please specify`
to 
`Please specify your choice of other location`

Keep in mind that this makes the label more verbose, and that could introduce another problem. It does though provide traceability to the area in which the choice exists without overly and redundantly explaining the singular choice.

For the checkbox, add a serial comma, to wit:
`Claiming as a firm, corporation, or state agency.`

The checkbox is within the context of further identifying the choice of **Other.** IMO, no further elucidation is required. 

For the field label on step 2, you could change
`Please specify`
to 
'Please specify your choice of other location`

Same explanation as above.

How's that work?

**Comment by fiorella-io** [comment]
With the new Burial redesign, I've addressed this issue. We are now removing "please" from our messaging, so "please" is not included. The copy of all error states in the form will be reviewed during our collaboration cycle. 
- [Relation to the deceased Veteran error page](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/1Ve1m9b)

There's still an outstanding question we'd like to ask our Burial SME about the "Claiming as a firm, corporation, or state agency" check box. To match the paper form, we've updated the options. We'll check to see if these new options will suffice. If not, we suggest adding "Firm / corporation / state agency" as another option.
